libs/examples/src/filter-field/filter-field-default-search-example/filter-field-default-search-example.ts

```diff
@@ -35,11 +35,7 @@ export class DtExampleFilterFieldDefaultSearch
       {
         name: 'Asia',
         defaultSearch: true,
-        suggestions: [
-          { name: 'Australia' },
-          { name: 'China' },
-          { name: 'India' },
-        ],
+        suggestions: [{ name: 'China' }, { name: 'India' }],
```

Comment:
Why did you remove Australia here?

Reply:
For obvious reasons. Australia belongs to Europe

Reply:
Would you also be able to provide a serious answer?